docs/products/clickhouse/index.rst

```diff
@@ -0,0 +1,40 @@
+ClickHouse :badge:`beta,cls=badge-secondary text-black badge-pill`
+==================================================================
+
+What is Aiven for ClickHouse?
```

Comment:
I believe we need to add "ClickHouse" to the list of the accepted terms

Reply:
Actually, I think it should be OK - at least it's not reported as an issue in the checks...

Reply:
That depends - at least it doesn't seem to raise any issues in the checks, but if we want to ensure the proper spelling, then it might be better to add "Clickhouse" to the rejected terms list?

Reply:
There is a warning which is raised :) It doesn't break and bring error, so I also almost missed it.

Reply:
Ah, in that case it's better to add it - that's now in ;)